optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/BenchmarkResultIO.java

```diff
@@ -108,9 +99,6 @@ protected PlannerBenchmarkResult readPlannerBenchmarkResult(File plannerBenchmar
             String benchmarkReportDirectoryName = plannerBenchmarkResultFile.getParentFile().getName();
             plannerBenchmarkResult = PlannerBenchmarkResult.createUnmarshallingFailedResult(
                     benchmarkReportDirectoryName);
-        } catch (XStreamException e) {
```

Comment:
Important.
Deleting this runtime exception XStreamException will remove the actual filename from the stacktrace message when a user has a bug in his solver configuration. That will leave the user clueless (= not good).